src/forwardmodels/Diffusion Approximation/DAcylinder_layered.jl

```diff
@@ -14,7 +14,7 @@
 Compute the steady-state fluence in an N-layered cylinder.
 
 # Arguments
-- `ρ`: source-detector separation in cylindrical coordinates (distance from middle z-axis of cylinder) (cm⁻¹)
+- `ρ`: source-detector separation or tuple of separations in cylindrical coordinates (distance from middle z-axis of cylinder) (cm⁻¹)
```

Comment:
I'm wondering if we should specify what types of the arguments should be explicitly. Like in this case rho can be an `AbstractFloat` or a 'Tuple{AbstractFloat}` or essentially a tuple of abstract floats. It might make more sense to have these as union of types in the argument so 
- `ρ::Union{AbstractFloat, Tuple{AbstractFloat}`:

Reply:
Yeah I think that would be a good idea for sure